.yarn/patches/web3-eth-accounts-npm-1.6.1-c95f31ca81.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/index.js b/lib/index.js
+index de2853c247b334e6b22cee42b3e597281c44efdc..787aecf6758228c092f1a3cd8097f64e76cc573d 100644
+--- a/lib/index.js
++++ b/lib/index.js
+@@ -344,7 +344,7 @@ function _handleTxPricing(_this, tx) {
+                             throw Error("Network doesn't support eip-1559");
+                         resolve({ gasPrice });
+                     }
+-                });
++                }).catch((ex) => reject(ex));
+             }
+         }
+         catch (error) {
```

Comment:
Please provide more details about this (in commit & PR message). In particular:
- Link to an issue reported on web3js repository
- What this patch does
- Why we need this, what will happen if we remove it (specific scenario).

Reply:
@ruzell22 +1 to the above, this should be explained in detail.
@outSH I wrote the code, here's the answer that @ruzell22 can copy paste into the PR description and the commit message: Without the catch block the rejection is an unhandled rejection that bubbles up to the top of the callstack where NodeJS itself catches it and then crashes the entire process. (used to be that it just logged a warning but since some of the newer versions it crashes which allows us to find these bugs in our code / library's code)

Reply:
@petermetz this is a nice learning! (Never knew that we can patch and include packages this way as well)

Reply:
@jagpreetsinghsasan Yeah, it's very useful when you need it. :-)

Reply:
@ruzell22 Please update the PR message as well. Also, if you could, please put the description in the point where you describe the changes (`Yarn patch to web3-eth-acconts@1.6.1 so that it does not crash nodejs process`) to be sure that this description you've provided applies to the patching.